app/Kconfig

```
@@ -325,6 +325,13 @@ config ZMK_RGB_UNDERGLOW_ON_START
 	bool "RGB underglow starts on by default"
 	default y
 
+config ZMK_RGB_UNDERGLOW_AUTO_OFF_IDLE
+	bool "Turn off RGB underglow when keyboard goes into idle state"
+
+config ZMK_RGB_UNDERGLOW_AUTO_OFF_USB
+	bool "Turn off RGB underglow when USB is disconnected"
+	depends on USB
```

```suggestion
	depends on USB_DEVICE_STACK
```

Needed w/ the Zephyr 3.0 update.

---

Rectified both issues